src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php

```diff
@@ -130,6 +130,17 @@ public function createAction(Request $request)
             if ($customerId = $result->getIdentifiableObjectId()) {
                 $this->addFlash('success', $this->trans('Successful creation.', 'Admin.Notifications.Success'));
 
+                $formWasSubmittedAsAjax = ($request->query->has('submitFormAjax'));
+                if ($formWasSubmittedAsAjax) {
```

Comment:
The parameter name talks itself
```suggestion
                if ($request->query->has('submitFormAjax')) {
```